packages/@azure/servicebus/data-plane/test/streamingReceiver.spec.ts

```diff
@@ -371,36 +371,36 @@ describe("Abandon message", function(): void {
     await testPeekMsgsLength(deadLetterClient, 0);
   }
 
-  it("Partitioned Queue: Multiple abandons until maxDeliveryCount.", async function(): Promise<
+  it("Partitioned Queue: Multiple abandons until maxDeliveryCount, abandon() retains message with incremented deliveryCount.", async function(): Promise<
     void
   > {
     await beforeEachTest(ClientType.PartitionedQueue, ClientType.PartitionedQueue);
     await testMultipleAbandons();
   });
 
-  it("Partitioned Topics and Subscription: Multiple abandons until maxDeliveryCount.", async function(): Promise<
+  it("Partitioned Topics and Subscription: Multiple abandons until maxDeliveryCount, abandon() retains message with incremented deliveryCount.", async function(): Promise<
```

Comment:
Let's replace all `Topics and Subscription` with just `Topic` or `Subscription`. It makes the title more verbose with no added benefit.

`sendSchedule.spec.ts` is the only file where we will use just `Topic`
All other test files are about receiving, so we can use just `Subscription`